Tests/NIOTests/MulticastTest.swift

```diff
@@ -165,14 +165,29 @@ final class MulticastTest: XCTestCase {
 
     func testCanJoinBasicMulticastGroupIPv4() throws {
         let multicastInterface = try assertNoThrowWithValue(self.interfaceForAddress(address: "127.0.0.1"))
+        guard multicastInterface.multicastSupported else {
+            // alas, we don't support multicast, let's skip
+            return
```

Comment:
I guess there is nothing like `assumeTrue(...)` ?

https://junit.org/junit4/javadoc/4.12/org/junit/Assume.html#assumeTrue(boolean)

Reply:
@normanmaurer no :(